packages/lib/src/components/internal/QRLoader/QRLoader.tsx

```diff
@@ -138,14 +138,14 @@ class QRLoader extends Component<QRLoaderProps, QRLoaderState> {
     };
 
     render({ amount, url, brandLogo, brandName, countdownTime, type, onActionHandled }: QRLoaderProps, { expired, completed, loading }) {
-        const { i18n, loadingContext } = useCoreContext();
+        const { i18n, loadingContext, resources } = useCoreContext();
```

Comment:
Perhaps we could use a custom hook to fetch the resources? It would get a bit cleaner 

```jsx
const { fetchImage } = useImages({ imageFolder: 'components/' }); // loadingContext could be  fetched within useResources, by using `useCoreContext` hook
...
return <img src={fetchImage('qrCode')} />
```